src/openforms/emails/utils.py

```diff
@@ -52,6 +52,7 @@ def send_mail_html(
     html_body: str,
     from_email: str,
     recipient_list: list[str],
+    cc: list[str] = None,
```

Comment:
```suggestion
    cc: list[str] | None = None,
```
the default value must always satisfy the type annotation too